bin/test_client.rb

```diff
@@ -159,6 +159,10 @@ def test_notification_response_data_type(notification, message_type)
 
   if message_type == 'precompiled_letter'
     field_should_not_be_nil(expected_fields_in_precompiled_letter_response, notification, 'send_precompiled_letter')
+    if notification.send(:"postage") != "first"
+      p "Postage should be set to 'first' for precompiled letter sending test. Right now it is set to #{notification[:postage]}"
```

Comment:
Hopefully this test won't ever fail 🤞, but if it does `notification[:postage]` will cause a `NoMethodError` - changing it to `notification.postage` instead will work.

Reply:
Ahh, I didn't know/forgot that this kind of construction also works in Ruby :D